pkg/lightning/backend/kv/session.go

```diff
@@ -283,14 +286,14 @@ func (*transaction) MayFlush() error {
 }
 
 type planCtxImpl struct {
-	*Session
+	*session
 	*planctximpl.PlanCtxExtendedImpl
 }
 
-// Session is a trimmed down Session type which only wraps our own trimmed-down
+// session is a trimmed down Session type which only wraps our own trimmed-down
 // transaction type and provides the session variables to the TiDB library
 // optimized for Lightning.
-type Session struct {
+type session struct {
```

Comment:
Here I make the original session private because not all fields in `sessionctx.Context` will be used. The public 
 struct `Session` will wrap this `session` and provide some necessary methods.

Reply:
please add this as code comment